<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HPRDmmCfg</class>
 <widget class="QDialog" name="HPRDmmCfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>254</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabCont">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 200);</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Serial">
    <attribute name="title">
     <string>Serial</string>
    </attribute>
    <widget class="QComboBox" name="cmbPort">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
    </widget>
    <widget class="QLabel" name="labComPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial port:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbBaud">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
     <property name="maxVisibleItems">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="labBaud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Baud rate:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbParity">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
     <property name="maxVisibleItems">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="labParity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labDbits">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Data bits:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbDbits">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
     <property name="maxVisibleItems">
      <number>2</number>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbSbits">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
     <property name="maxVisibleItems">
      <number>3</number>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbFlow">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
    </widget>
    <widget class="QLabel" name="labSbits">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop bits:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labFlow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Flow control:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="GPIB">
    <attribute name="title">
     <string>GPIB</string>
    </attribute>
    <widget class="QLabel" name="labGpibAddr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Instrument GPIB Address:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbGpibAddr">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
     <property name="maxVisibleItems">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="labID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Contoller ID String:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIDStr">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Logging">
    <attribute name="title">
     <string>Log</string>
    </attribute>
    <widget class="QLabel" name="labLogE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Log enable:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labLogF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Log file:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLogEnabled">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLogFile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labLogInt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Log Interval:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLogInt">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbLogIntUnit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="StartUp">
    <attribute name="title">
     <string>Start</string>
    </attribute>
    <widget class="QLabel" name="labSprof">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>StartUp Profile:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbSp">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btnSprofCfg">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="labDispStyle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Display Style:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbDispStyle">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ExΩ</string>
    </attribute>
    <widget class="QLabel" name="labIntRes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Internal Resistor Value:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkExoReal">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labCalcActual">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate actual resistance:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spnExoR1Val">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="minimum">
      <double>9.500000000000000</double>
     </property>
     <property name="maximum">
      <double>10.500000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000010000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="labExoInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>351</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;By default, in Extended Ohms mode, the HP3478 shows the combined resistance of the internal 10MΩ resistor (R1) and the resistance of the component under test (R2) in parallel. Checking this option will apply the formula to calculate and display the actual resistance of the component under test.&lt;/p&gt;&lt;p&gt;With nothing connected to the meter, the value of the internal resistor alone will be shown. For accurate results, this should be entered into the ‘Internal Resistor Value’ field.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Cont</string>
    </attribute>
    <widget class="QLabel" name="labSndEnable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable buzzer:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkEnableBeep">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labCntThresh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Continuity threshold:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnContThreshold">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 240);</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>199</number>
     </property>
     <property name="value">
      <number>55</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="labCntOhms">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ohms</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Temp">
    <attribute name="title">
     <string>Temp</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="CalData">
    <attribute name="title">
     <string>Cal</string>
    </attribute>
    <widget class="QTextBrowser" name="txtCalData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>361</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReadCal">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Backup Calibration Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnWriteCal">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore Calibration Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_Disclaim">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(200, 0, 0);</string>
     </property>
     <property name="text">
      <string>Disclaimer: Use this feature at your own risk!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HPRDmmCfg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HPRDmmCfg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
